site/package.json

```diff
@@ -30,12 +30,43 @@
     "gulp-rev-delete-original": "^0.2.3",
     "gulp-stylus": "^2.7.0",
     "gulp-uglify-es": "^1.0.4",
-    "gulp-webserver": "^0.9.1"
+    "gulp-webserver": "^0.9.1",
+    "husky": "^1.2.0",
+    "lint-staged": "^8.1.0",
+    "xo": "github:aslafy-z/xo#fix/lint-staged-absolute"
```

Comment:
Note that I'm using a fork, see here: https://github.com/xojs/xo/pull/355 https://github.com/xojs/xo/pull/279

Reply:
https://github.com/swapagarwal/swag-for-dev/pull/169 would solve it as we would be able to use the `relative` option of `lint-staged`.

Reply:
Does that mean you want to wait for that PR?

Reply:
It would be easier, or we can merge using my fork until we change the website root.
@itaisteinherz are you willing to finish it soon? I can rebase your work so we can go through if you want

Reply:
I’m pretty sure that #169 is ready to be merged 👍🏻
I’ll be away from my computer for the next 3 days, so if you have anything you’d like to change feel free to push that to the PR. I can also talk on GitHub / Gitter if you’d like me to give feedback on anything.

Reply:
Nice! I've updated #169, it needed a final rebase.
This is going through! :tada: